tools/mlir-proto-opt/mlir-proto-opt.cpp

```diff
@@ -23,8 +24,11 @@ using namespace mlir;
 
 int main(int argc, char **argv) {
   llvm::InitLLVM y(argc, argv);
+  registerAllPasses();
   registerOutsideOfDialectRegistry();
 
+  // TODO: Find a way to register test passes here.
```

Comment:
Add forward declarations above and call them here?

Reply:
`mlir-opt.cpp` has a long list of test passes to be registered. It seems impractible to copy the entire list. It would go out-of-sync pretty fast. Maybe there's a way to put these into a separate function in MLIR. Something like `registerAllTestPasses`. It seems like we don't depend on any MLIR test passes in the Sandbox yet, so we could keep the TODO for the moment.

Reply:
We _shouldn't_ depend on MLIR test passes, that's why they are hidden. So I wouldn't even add a TODO here :)